indico/modules/auth/forms.py

```diff
@@ -100,7 +100,7 @@ class MultipassRegistrationForm(SyncedInputsMixin, IndicoForm):
 
 
 class LocalRegistrationForm(RegistrationForm):
-    email = EmailField(_('Email address'))
+    email = EmailField(_('Email address'), [DataRequired(), _check_existing_email])
```

Comment:
why? this field is always supposed to be disabled (and thus never in the submitted data) since we only get to that page after validating the email - so it's never editable.